monkey/monkey_island/cc/ui/src/components/configuration-components/PluginSelectorTemplate.tsx

```diff
@@ -0,0 +1,48 @@
+import {ObjectFieldTemplateProps} from '@rjsf/utils';
+import React, {useState} from 'react';
+import {Dropdown} from 'react-bootstrap';
+
+const PluginSelector = (props) => {
+  let selectOptions = [];
+  for (const [name, plugin] of Object.entries(props.plugins)) {
+    selectOptions.push(
+      <Dropdown.Item onClick={() => props.onClick(name)}
+                     eventKey={`plugin['title']`}>
+                     {plugin['title']}</Dropdown.Item>
+  )}
+
+  return (
+    <Dropdown>
+      <Dropdown.Toggle variant="success" id="dropdown-basic">
+        Select a plugin
+      </Dropdown.Toggle>
+      <Dropdown.Menu>
+        {selectOptions}
+      </Dropdown.Menu>
+    </Dropdown>
+  )
+}
+
+
+export default function PluginSelectorTemplate(props: ObjectFieldTemplateProps) {
+
+  function getPluginDisplay(plugin, allPlugins){
+    let selectedPlugin = allPlugins.filter((pluginInArray) => pluginInArray.name == plugin)
+    if(selectedPlugin.length === 1){
```

Comment:
It's probably impossible for this to be a length other than 0 or 1, but we can code a bit more defensively:
```suggestion
    if(selectedPlugin.length > 0){
```

Reply:
I'm not sure how much of this code will remain, let's re-consider this once the plugin selector is more fleshed-out